<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::WidgetWidth</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuFont_Size">
     <property name="title">
      <string>Font Size</string>
     </property>
     <addaction name="action12pt"/>
     <addaction name="action16pt"/>
     <addaction name="action24pt"/>
     <addaction name="action32pt"/>
    </widget>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <widget class="QMenu" name="menuLight">
      <property name="title">
       <string>Light</string>
      </property>
      <addaction name="actionSelect"/>
      <addaction name="separator"/>
      <addaction name="actionSet_as_Default"/>
     </widget>
     <widget class="QMenu" name="menuSilver_Fox">
      <property name="title">
       <string>Silver Fox</string>
      </property>
      <addaction name="actionSelect_2"/>
      <addaction name="separator"/>
      <addaction name="actionSet_as_Default_2"/>
     </widget>
     <widget class="QMenu" name="menuDark">
      <property name="title">
       <string>Dark</string>
      </property>
      <addaction name="actionSelect_3"/>
      <addaction name="separator"/>
      <addaction name="actionSet_as_Default_3"/>
     </widget>
     <widget class="QMenu" name="menuMatrix">
      <property name="title">
       <string>Matrix</string>
      </property>
      <addaction name="actionSelect_4"/>
      <addaction name="separator"/>
      <addaction name="actionSet_as_Default_4"/>
     </widget>
     <widget class="QMenu" name="menuBeach_Boi">
      <property name="title">
       <string>Beach Boi</string>
      </property>
      <addaction name="actionSelect_5"/>
      <addaction name="separator"/>
      <addaction name="actionSet_as_Default_5"/>
     </widget>
     <widget class="QMenu" name="menuChickadee">
      <property name="title">
       <string>Chickadee</string>
      </property>
      <addaction name="actionSelect_6"/>
      <addaction name="separator"/>
      <addaction name="actionSet_as_Default_6"/>
     </widget>
     <widget class="QMenu" name="menuBlack_n_Gold">
      <property name="title">
       <string>Black n Gold</string>
      </property>
      <addaction name="actionSelect_7"/>
      <addaction name="separator"/>
      <addaction name="actionSet_as_Default_7"/>
     </widget>
     <widget class="QMenu" name="menuPink">
      <property name="title">
       <string>Pink</string>
      </property>
      <addaction name="actionSelect_8"/>
      <addaction name="separator"/>
      <addaction name="actionSet_as_Default_8"/>
     </widget>
     <widget class="QMenu" name="menuSlate">
      <property name="title">
       <string>Slate</string>
      </property>
      <addaction name="actionSelect_9"/>
      <addaction name="separator"/>
      <addaction name="actionSet_as_Default_9"/>
     </widget>
     <widget class="QMenu" name="menuWheat">
      <property name="title">
       <string>Wheat</string>
      </property>
      <addaction name="actionSelect_10"/>
      <addaction name="separator"/>
      <addaction name="actionSet_as_Default_10"/>
     </widget>
     <addaction name="menuLight"/>
     <addaction name="menuSilver_Fox"/>
     <addaction name="menuDark"/>
     <addaction name="menuMatrix"/>
     <addaction name="menuBeach_Boi"/>
     <addaction name="menuChickadee"/>
     <addaction name="menuBlack_n_Gold"/>
     <addaction name="menuPink"/>
     <addaction name="menuSlate"/>
     <addaction name="menuWheat"/>
    </widget>
    <addaction name="menuFont_Size"/>
    <addaction name="menuTheme"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSelectAll"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit_Program">
   <property name="text">
    <string>Exit Program</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="action12pt">
   <property name="text">
    <string>12pt</string>
   </property>
  </action>
  <action name="action16pt">
   <property name="text">
    <string>16pt</string>
   </property>
  </action>
  <action name="action24pt">
   <property name="text">
    <string>24pt</string>
   </property>
  </action>
  <action name="action32pt">
   <property name="text">
    <string>32pt</string>
   </property>
  </action>
  <action name="actionSet_as_Default">
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionSet_as_Default_2">
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionSet_as_Default_3">
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionSet_as_Default_4">
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionSet_as_Default_5">
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionSet_as_Default_6">
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionSet_as_Default_7">
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionSet_as_Default_8">
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionSet_as_Default_9">
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionSet_as_Default_10">
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionSelect_2">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionSelect_3">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionSelect_4">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionSelect_5">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionSelect_6">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionSelect_7">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionSelect_8">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionSelect_9">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionSelect_10">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>

